casa/OS/HostInfo.cc

```diff
@@ -271,5 +291,8 @@ ptrdiff_t HostInfo::swapFree( )    { return -1; }
 namespace casacore { //# NAMESPACE CASACORE - BEGIN
 
 HostMachineInfo *HostInfo::info = 0;
+int HostInfo::resources_memory = 0;
+int HostInfo::resources_memfrac = 0;
+int HostInfo::resources_cores = 0;
```

Comment:
When the value is read from .aipsrc is comes as int, and then is casted to ptrdiff_t so I keep that pattern